### DISCUSSION TITLE
Monthly ESMValTool community meeting March 2023

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#84] [Meetings]
Dear @ESMValGroup/esmvaltool-developmentteam,

The next monthly ESMValTool community meeting will be end of March. The result of the poll is that the meeting will take place on

Tuesday, March 28 at 10 AM CET

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

**Format and agenda**

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we choose some topics to talk about in more detail.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just [create a new GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new) about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in #73

**Comment by ehogan** [comment]
Hi all 👋 The RSE Conference 2023 ([RSECon23](https://rsecon23.society-rse.org/)) will be held in Swansea during the week starting Monday September 4. The themes this year are:

- Working with and as Researchers
- Open Science and Open Source
- Working with Industry and Commercialisation
- Tooling for Research Software Engineering

Since "Open Source" is listed, I wondered whether it would be worth contributing one or more ESMValTool-related:

- [Talks](https://rsecon23.society-rse.org/talks/)
- [Audience-led panels](https://rsecon23.society-rse.org/audience-led-panels/)
- [Presenter-led panels](https://rsecon23.society-rse.org/presenter-led-panels/)
- [Walkthroughs](https://rsecon23.society-rse.org/walkthroughs/)
- [Workshops](https://rsecon23.society-rse.org/workshops/)
- [Hackathons](https://rsecon23.society-rse.org/hackathons/)
- [Posters](https://rsecon23.society-rse.org/posters/)

See [RSECon23: Call for submissions 2023](https://rsecon23.society-rse.org/submissions/) for more details. Call for submissions closes on Tuesday, April 25, 2023.

Is anyone planning on attending and would anyone be interested in contributing something?

**Reply by bouweandela** [reply]
@SarahAlidoost Maybe this could interest you since you gave this [talk](https://wosss.org/wosss21/S2-FakherehAlidoost) at WoSSS21?

**Comment by bouweandela** [comment]
@glpotter Let me know that he would be interested to hear about progress on the CFSR CMORizer. Any news on this topic?

**Comment by bouweandela** [comment]
@rswamina Let me know that she will join the meeting today, but may be a bit later.

**Comment by bouweandela** [comment]
Updates
- @rbeucher - joined from [ACCESS-NRI](https://www.access-nri.org.au/)
- @catherinehardacre  - diagnostics on aerosoll optical depth
- @katjaweigel - working on IPCC figures
- @axel-lauer - Working on diagnostics to analyse reanalises and CMORizers that uses GRIB files, potentially useful for ERA5 too?
- @LisaBock - doing outreach for the release
- @mo-tgeddes - working on moving autoasess into ESMValTool
- @remi-kazeroni - currently release manager of v2.8
- @zklaus - working on impact related topics with @rswamina and preparing an ESMValTool workshop at SMHI and new proposal to acquire new funding
- @ehogan - expects to be involved more with the community in the coming months
- @sloosvel - maintained recipes for the new release, planning more fixes for CORDEX data and derived variables for the next release
- @tepmo42 - working on Miles blocking recipe https://github.com/ESMValGroup/ESMValTool/discussions/3092 with their own data. Interested in using HadGEM model data, we talked about potentially supporting this model directly within the ESMValTool without prior CMORization, similar to the models described in [Schlund et. al. 2023](https://gmd.copernicus.org/articles/16/315/2023/gmd-16-315-2023-discussion.html)/[the documentation](https://docs.esmvaltool.org/en/latest/input.html#datasets-in-native-format).
- @rswamina - preparing a tutorial in May and an advanced tutorial in September for people who would like to add their datasets (add a CMORizer) - Interested in using ESMValTool with jupyter notebooks
- @bouweandela - working on improved support for large datasets in the upcoming v2.9 release
- @valeriupredoi - second release manager for v2.8

**Comment by bouweandela** [comment]
Topics that were discussed:

- Workshop: see https://github.com/ESMValGroup/Community/discussions/87 for details, date will be finalized soon.
- Release: [ESMValCore v2.8](https://github.com/ESMValGroup/ESMValCore/discussions/1924) released, [ESMValTool v2.8](https://github.com/ESMValGroup/ESMValTool/discussions/3055) expected soon. See the linked discussions for more information. Thanks to everyone who contributed!
- Relevant links for using ESMValTool from the Jupyter notebook:
  - [short course](https://github.com/ESMValGroup/EGU22-short-course/)
  - [example notebooks](https://github.com/ESMValGroup/ESMValCore/tree/main/notebooks)
  - [configuring a run from a notebook](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.config.html) (useful in combination with the [`resume_from`](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/run.html) configuration option when developing a recipe)
  - [running a recipe from a notebook](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.experimental.recipe.html#running-a-recipe)
  - @zklaus asked how do you develop a diagnostic in a notebook and then turn it into a recipe? If you have experience with this, please share them in a comment below or e.g. in this issue https://github.com/ESMValGroup/ESMValCore/issues/1843.
- [List of supported datasets](https://docs.esmvaltool.org/en/latest/input.html#supported-datasets-for-which-a-cmorizer-script-is-available) that are not available from ESGF
- CFSR CMORizer work in progress in https://github.com/ESMValGroup/ESMValTool/pull/2940. 
- RSE conference, @rswamina suggested discussing this with the @ESMValGroup/userengagementteam 
- If you're interested in hosting the monthly community meetings, e.g. for a year, please let @bouweandela know. The typical time investment is about 3 hours per month.

Thanks for joining everyone!

**Comment by bouweandela** [comment]
If you're coming to EGU, catch up with the other ESMValTool community members at our splinter meeting: #90.
